src/course/syllabus/pre-app-5/project.md

```diff
@@ -0,0 +1,49 @@
+Your project for this week is to continue working on your [digital agency website](../../pre-app-4/project).
+
+You'll add a way to get in touch if they are interested in working with you. When the user clicks submit on the form, their request should be added to a visible backlog on the page. The backlog does not need to be preserved when the user refreshes the page.
```

Comment:
I like this idea—my only concern is that it doesn't easily break down into separate async tasks. It's just one big feature, so might be difficult for them to manage working on separately

Reply:
That's a good point - I guess only one person would be practising the template learnings and they rely on each other to finish different parts. 

I wonder if adding two of these kinds of features would be worthwhile or result in doubling the workload. We could ask them to arrange to code the template part together.